influxdb/files/influxdb.conf

```diff
@@ -187,7 +187,11 @@ reporting-disabled = false
 [http]
   enabled = true
   bind-address = "{{ server.http.bind.address }}:{{ server.http.bind.port }}"
+  {%- if server.admin_user is defined %}
```

Comment:
I'm fine with it.
It's about having metadata structure well designed from the beginning, because then it's difficult to change it while maintaining backward compatibility.